docs/tutorials/setup-opentelemetry.md

```diff
@@ -24,16 +24,18 @@ yarn --cwd packages/backend add @opentelemetry/sdk-node \
 
 ## Configure
 
-In your `packages/backend/src` folder, create an `instrumentation.ts` file.
+In your `packages/backend` folder, create an `instrumentation.ts` file.
 
 ```typescript
-import { NodeSDK } from '@opentelemetry/sdk-node';
-import { ConsoleSpanExporter } from '@opentelemetry/sdk-trace-node';
-import { getNodeAutoInstrumentations } from '@opentelemetry/auto-instrumentations-node';
-import {
+const { NodeSDK } = require('@opentelemetry/sdk-node');
+const { ConsoleSpanExporter } = require('@opentelemetry/sdk-trace-node');
+const {
+  getNodeAutoInstrumentations,
+} = require('@opentelemetry/auto-instrumentations-node');
+const {
   PeriodicExportingMetricReader,
   ConsoleMetricExporter,
-} from '@opentelemetry/sdk-metrics';
+} = require('@opentelemetry/sdk-metrics');
```

Comment:
these are missing typehints now I believe :/ Is there a way to use require and maintain typehints?

Reply:
Honestly, I am not sure. I am not that good at typescript/javascript. I'd like to get this PR merged, maybe someone in the future will have a better idea for this whole setup, but for now I think it will do just fine. ESLint still works, so that is a plus